lib/output.js

```diff
@@ -187,12 +187,15 @@ function OutputStream(options) {
             }
             might_need_space = false;
         }
-        var a = str.split(/\r?\n/), n = a.length - 1;
-        current_line += n;
-        if (n == 0) {
-            current_col += a[n].length;
-        } else {
-            current_col = a[n].length;
+        var a = str.split(/\r?\n/), len = a.length;
+        if (len > 0) {
+            var n = len - 1;
+            current_line += n;
+            if (n == 0) {
+                current_col += a[n].length;
+            } else {
+                current_col = a[n].length;
+            }
         }
```

Comment:
In Internet Explorer 8 with the `beautify` option equal to `true` the following error occurs:

**Unable to get property 'length' of undefined or null reference**

	var a = str.split(/\r?\n/), n = a.length - 1;
	current_line += n;
	if (n == 0) {
		current_col += a[n].length;
	} else {
		current_col = a[n].length;
	____^
	}

My patch fixes this error. It would be nice to merge it.